openwisp_controller/config/base/template.py

```diff
@@ -165,6 +165,12 @@ def get_context(self):
         context.update(super().get_context())
         return context
 
+    def get_system_context(self):
+        system_context = self.get_context()
+        for key in self.default_values:
+            system_context.pop(key, None)
```

Comment:
use `delete system_context[key]`, `pop` makes sense only when the value has to be retrieved